src/styleguide/_patterns/04-templates/00-page-layouts/01-error-page.mustache

```diff
@@ -1,6 +1,6 @@
 {{> organisms-page-header }}
 <main class="grav-o-container">
-  <section class="grav-o-section--centered">
+  <section class="grav-o-section grav-o-section--centered">
```

Comment:
Rather than use `grav-o-section--centered` for the text centering, you could switch to using the `grav-u-text-centered` utility class that @ThePeach recently added. Less redundancy in our CSS that way.

(Or he should update the homepage here to be a `grav-o-section` and kill that utility class? I'll leave it to you two to duke it out!)

Reply:
Utility class method suits me, I think we could do with using a few more of them IMO.